app/controllers/overrides/sessions_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Overrides::SessionsController < DeviseTokenAuth::SessionsController
```

Comment:
I tried with a user registered with the web client. The Rspec tests pass as normal. The exception is gone on Postman. It shows this message 
```
{"errors":["Invalid login credentials. Please try again."]}
```
However the CSRF error is still there in terminal and says that it is not authorized.  
The message in the terminal is 
```
Started POST "/api/v1/auth/sign_in" for ::1 at 2016-07-26 14:37:36 +0200
Processing by Overrides::SessionsController#create as */*
Completed 401 Unauthorized in 1ms
```